src/main/java/build/buildfarm/common/DigestPath.java

```diff
@@ -0,0 +1,30 @@
+// Copyright 2024 The Buildfarm Authors. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package build.buildfarm.common;
+
+import build.buildfarm.v1test.Digest;
+import java.nio.file.Path;
+import lombok.Data;
+
+@Data
+public final class DigestPath {
```

Comment:
How about a Java 17 "Record"? https://docs.oracle.com/en/java/javase/17/docs/api/java.base/java/lang/Record.html

Reply:
Neat. Done. We should switch a bunch of things over to this.